packages/java/pom.xml

```diff
@@ -4,11 +4,40 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.aneo</groupId>
+  <groupId>fr.aneo</groupId>
   <artifactId>armonik-java</artifactId>
   <description>GRPC java binding for the Armonik orchestrator API</description>
   <version>0.1.0</version>
 
+	<name>armonik-java</name>
+	<url>https://github.com/aneoconsulting/ArmoniK.Api</url>
+
+	<developers>
```

Comment:
I think that we should add Hassan who start it even if he is not anymore with aneo.

Reply:
Yes, I will add him.